minecraft_servers/badhub/manifest.json

```diff
@@ -0,0 +1,46 @@
+{
+  "server_name": "badhub",
+  "nice_name": "BadHub",
+  "direct_ip": "mc.badhub.cz",
+  "server_wildcards": [
+    "%.badhub.cz"
+  ],
+  "supported_languages": [
+     "cz",
+     "sk"
+  ],
+  "social": {
+    "web": "https://www.badhub.cz/",
+    "web_shop": "https://vip.vagonbrei.eu/link/LAkWETe4mc",
+    "tiktok": "mc.badhub.cz",
+    "instagram": "badhub.cz",
+    "discord": "https://discord.badhub.cz"

```

Comment:
Custom invite links are reserved for partners only; use a normal discord.gg/[CODE] link instead